tests/services/test_sort.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Sort tests."""
+
+
+import pytest
+
+from invenio_rdm_records.services import RDMRecordService
+
+# @pytest.fixture()
+# def service(appctx):
+#     """Service fixture."""
+#     return RDMRecordService()
+#     # RDMRecordService(
+#     #     config=app.config.get(RDMRecordService.config_name),
+#     # )
+
+@pytest.fixture()
+def service(app, location):
+    """Service fixture."""
+    return RDMRecordService()
+
+
+def test_sort_by_versions(
+        service, superuser_identity, minimal_record):
+    # Create version 1
+    draft = service.create(superuser_identity, minimal_record)
+    record = service.publish(draft.id, superuser_identity)
+    # Create version 2
+    draft = service.new_version(record.id, superuser_identity)
+    record_2 = service.publish(draft.id, superuser_identity)
+    # Create version 3
+    draft = service.new_version(record_2.id, superuser_identity)
+    record_3 = service.publish(draft.id, superuser_identity)
+    # NOTE: we swap version 2 and 3, so that "newest" order is different
+    #       than "versions" order
+    # Swap version 2 and 3 - have to reach into the datalayer to do so
+    record_2._record.versions.index = 3
```

Comment:
Can't assign. Do I reach deeper into the data record or do I modify the systemfield?